docs/spl-docs/evaluation-functions/mathematical-functions.md

```diff
@@ -0,0 +1,441 @@
+| eval log_visits = ln(total_visits)
+| fields date, total_visits, log_visits
+```
+
+**Explanation:**
+1. The `stats` command calculates the total number of website visits per day.
+2. The `eval` command calculates the natural logarithm of the total visits using `ln(total_visits)`.
+
+This allows the user to analyze the log-transformed data to identify exponential growth patterns in website traffic.
+
+## log(\<num>, \<base>)
+
+This function returns the logarithm of a number using a base. 
+
+### Usage
+
+The `<num>` argument can be the name of a numeric field or a numeric literal.
+
+The `<base>` is optional, and if omitted the log function uses base 10.
+
+### Example
+
+The following example returns the logarithm of the values of the `number` field, using base 2.
+
+```
+... | eval num=log(number,2)
+```
+
+The following example returns the logarithm of the numeric literal `100000`, using base 10.
+
+```
+... | eval num=log(100000)
+```
+
+### Use-Case Example
+
+**Analyze order of magnitude in financial transactions**
+
+**Problem:** A user wants to categorize financial transactions based on their order of magnitude to identify large, medium, and small transactions for risk assessment and reporting purposes.
+
+**Solution:** The `log()` command can be used to calculate the logarithm of transaction amounts, making it easier to categorize them based on their magnitude.
+
+```
+index=financial_transactions
+| eval log_amount = log(transaction_amount, 10)
+| eval category = case(log_amount >= 5, "Large", log_amount >= 3, "Medium", log_amount < 3, "Small")
+| fields transaction_id, transaction_amount, log_amount, category
+```
+
+**Explanation:**
+1. The `eval` command calculates the logarithm (base 10) of the transaction amounts using `log(transaction_amount, 10)`.
+2. The `category` field is created using a `case` statement to categorize transactions based on their logarithmic value.
+
+This allows the user to categorize transactions efficiently, aiding in risk assessment and detailed financial reporting.
+
+## pow(\<num>, \<exp>)
+
+This function returns a number to the power of the exponent.
+
+### Usage
+
+The `<num>` argument can be the name of a numeric field or a numeric literal.
+
+The `<exp>` argument is the exponent.
+
+### Example
+
+The following example calculates the area of a circle, which is `pi()` multiplied by the radius to the power of 2.
+
+```
+... | eval area_circle=pi()*pow(radius,2)
+```
+
+### Use-Case Example
+
+**Calculate future investment value with compound interest**
+
+**Problem:** A user wants to calculate the future value of an investment based on compound interest. The formula for compound interest is given by `Future_Value = Principal * (1 + annual_interest_rate/times_compounded_per_year)^(times_compounded_per_year * years)`.
+
+**Solution:** The `pow()` command can be used to compute `(1 + annual_interest_rate/times_compounded_per_year)^(times_compounded_per_year * years)`.
```

Comment:
nit: these equations don't display well for me. Use code blocks instead of inline code
<img width="951" alt="image" src="https://github.com/siglens/siglens-docs/assets/20426590/1e5e5e45-602b-4638-be81-678b1bcadeaf">